core/src/test/java/org/apache/accumulo/core/spi/balancer/HostRegexTableLoadBalancerReconfigurationTest.java

```diff
@@ -108,7 +107,7 @@ public void testConfigurationChanges() {
     // getOnlineTabletsForTable
     UtilWaitThread.sleep(3000);
     this.balance(new BalanceParamsImpl(Collections.unmodifiableSortedMap(allTabletServers),
-        migrations, migrationsOut, DataLevel.USER, tables));
+        migrations, migrationsOut, "USER", tables));
```

Comment:
```suggestion
        migrations, migrationsOut, DataLevel.USER.name(), tables));
```
I would still use the enum here for all the tests instead of the hardcoded string value. this matches what the Manager does as well in this change (it is just calling .name() on the datalevel)

Reply:
Removed all hardcoded string values in favor of the `.name()` method.